config.go

```diff
@@ -128,7 +128,7 @@ func (cfg *Config) SetProfiler(profiler ProfilingStrategy) error {
 	return nil
 }
 
-// Enables compiled code caching for this `Config` using the default settings
+// CacheConfigLoadDefault enables compiled code caching for this `Config` using the default settings
 // configuration can be found.
```

Comment:
So, the second line here is a hold-over from the original docstring but we can take the opportunity now to clean it up.

Pretty sure if we delete `configuration can be found` it still reads the same.